src-tauri/src/platform_specific/linux.rs

```diff
@@ -1,8 +1,80 @@
 // Linux specific code
 
 use regex::Regex;
+use std::env;
```

Comment:
I wouldn't namespace (or whatever it's called) a function if it's just called once
```suggestion
```